Google.Cloud.Spanner.NHibernate.Samples/Snippets/SchemaExportSample.cs

```diff
@@ -0,0 +1,67 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using NHibernate.Tool.hbm2ddl;
+using System;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Spanner.NHibernate.Samples.Snippets
+{
+    /// <summary>
+    /// Sample showing how to use <see cref="SchemaExport"/> with Google Cloud Spanner.
+    ///
+    /// The standard NHibernate <see cref="SchemaExport"/> is not compatible with the DDL dialect of Cloud Spanner.
+    /// The Cloud Spanner driver therefore comes with its own implementation in <see cref="SpannerSchemaExport"/>. The
+    /// SpannerSchemaExport generates DDL that is compatible with Cloud Spanner, and automatically executes multiple DDL
+    /// statements as one batch, which significantly improves execution speed for large database schemas.
+    /// 
+    /// Run from the command line with `dotnet run SchemaExport`
+    /// </summary>
+    public static class SchemaExportSample
+    {
+        public static async Task Run(SampleConfiguration configuration)
+        {
+            using var session = configuration.SessionFactory.OpenSession();
+            
+            // The sample runner automatically creates the data model that is required for the entities that are used
+            // by the samples in this project. The following query gets the number of tables currently in the database.
+            // The SQL script that is used to create the data model can be found in `SampleModel/SampleDataModel.sql`.
+            var cmd = session.Connection.CreateCommand();
+            cmd.CommandText =
+                "SELECT COUNT(*) FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_CATALOG='' AND TABLE_SCHEMA=''";
+            var numberOfTablesBeforeDropAndRecreate = (long) await cmd.ExecuteScalarAsync();
+            
+            // Create a SpannerSchemaExporter and use this to drop and re-create the database schema from the entity
+            // model. This should result in the exact same data model as the SQL statements in `SampleDataModel.sql`.
+            // The mapping model must include all metadata for the schema, such as which columns are not-nullable, the
+            // length of STRING columns, etc.
+            // Tables are created in the order that they are added to the mapping. Tables are dropped in the opposite
+            // order. This means that if the model includes interleaved tables, the parent table must be added BEFORE
+            // the child table. See SampleConfiguration.cs for an example.
+            var exporter = new SpannerSchemaExport(configuration.Configuration);
+            // This will automatically execute a drop-and-recreate script and print the statements that are executed to
+            // the console (StdOut).
+            await exporter.CreateAsync(true, true);
```

Comment:
If you can do some inline comments to explain what the `true, true` are that would be great.
```suggestion
            await exporter.CreateAsync(true /*useStdOut*/, true /*execute*/);
```